build.sbt

```diff
@@ -11,6 +11,7 @@ scalacOptions ++= Seq(
   "-feature",
   "-deprecation",
   "-unchecked",
+  "-target:jvm-1.8",
```

Comment:
Building for Java 8 has been necessary in multiple environments. And we haven't seen any downsides so far. Let us know if you see any!